### ISSUE TITLE
tentia: URL format

### ISSUE THREAD

**Issue by tjreo** [jeena/Bungloo#55]
It would be great for Tentia to register a URL handler so that it could be passed something like tentia:///tjreo.tent.is/3j8dnl and to have it open in conversation view.

**Comment by jeena** [comment]
Yes actually I was thinking about implementing one of the tent URL schemas proposed here: https://github.com/tent/tent.io/wiki/Tent-URL-Scheme

**Comment by poweruser82** [comment]
I like the alternate scheme more.
